fsspec/implementations/http.py

```diff
@@ -231,21 +232,21 @@ async def _cat_file(self, url, start=None, end=None, **kwargs):
         return out
 
     async def _get_file(
```

Comment:
I thunk this one was right! 
It says, "get file from remote to local", and should be the revers of put_file. That's how we have it in spec.

Reply:
Ah, missed that! Thanks for the correction.